File: src/main/java/kr/co/lunatalk/domain/category/dto/response/CategoryCreateResponse.java
package kr.co.lunatalk.domain.category.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;
import kr.co.lunatalk.domain.category.domain.Category;

public record CategoryCreateResponse(
	@Schema(description = "추가한 categoryId")
	Long categoryId
) {
	public static CategoryCreateResponse of(Category category) {
		return new CategoryCreateResponse(category.getId());
	}
}

File: lunatalk-domain/src/main/kotlin/kr/co/lunatalk/domain/category/dto/response/CategoryCreateResponse.kt
package kr.co.lunatalk.domain.category.dto.response

import io.swagger.v3.oas.annotations.media.Schema
import kr.co.lunatalk.domain.category.domain.Category

data class CategoryCreateResponse(
    @Schema(description = "추가한 categoryId")
    val categoryId: Long?
) {
    companion object {
        fun of(category: Category): CategoryCreateResponse {
            return CategoryCreateResponse(category.id)
        }
    }
}
